python/doc/theory/reliability_sensitivity/sensitivity_hsic.rst

```diff
@@ -0,0 +1,293 @@
+.. _sensitivity_hsic:
+
+Sensitivity analysis using Hilbert-Schmidt Indepencence Criterion (HSIC)
+------------------------------------------------------------------------
+
+Introduction
+~~~~~~~~~~~~
+
+The Hilbert-Schmidt Indepencence Criterion deals with analyzing the influence that the random vector
+:math:`\vect{X} = \left( X^1,\ldots,X^{d} \right)` has on a random variable
+:math:`Y`, which is being studied for uncertainty. Here, we attempt to evaluate the influence
+through the dependence between the two random variables :math:`Y` and :math:`X^i`.
+In practice, we compute the dependence between :math:`Y` and :math:`X^i` as the
+distance between the joint distribution :math:`P_{Y,X^i}` and the 
+product of the marginal distributions :math:`P_{X^i}P_{Y}`. 
+
+In the following paragraphs, we consider an independent and identically distributed
+learning sample of size :math:`n`, which can for instance be obtained through
+Monte Carlo sampling or real life observations:
+
+.. math::
+
+    \left(\mathbf{X}^{(j)}, Y^{(j)}\right)_{(1\leq j\leq n)} = \left(X_1^{(j)}, X_2^{(j)}, \dots, X_d^{(j)}, Y^{(j)}\right)_{(1\leq j\leq n)}
+
+where :math:`\mathbf{X}` and :math:`Y` respectively follow :math:`P_{\mathbf{X}}` and :math:`P_{Y}`.
+In many cases, only :math:`\mathbf{X}` is sampled, while :math:`P_{Y}` is obtained
+as the output of a computer code: :math:`Y^{(j)} = \mathcal{M}\left(X_1^{(j)}, X_2^{(j)}, \dots, X_d^{(j)}\right)`
```

Comment:
The computer code is not "caligraphic M", it should denoted by "g" (see for example https://openturns.github.io/openturns/master/theory/reliability_sensitivity/importance_form.html). The letter "g" should be downcase, because it is a deterministic computer code, and not "G", which may imply that it is random. It should not be a caligraphic letter, which should be used for sets (a function is not a set). The letter "g" is used in most tutorials in OpenTURNS and our internal training and should be used consistently.

I admit that this convention is unconsistently used in the current doc. For example, the page https://openturns.github.io/openturns/master/theory/meta_modeling/kriging.html uses "M", which is a bug, unrelated to the present document.